test/cmd/newapp.sh

```diff
@@ -376,9 +377,23 @@ os::cmd::expect_success 'oc delete imagestreamtag ruby:2.3'
 os::cmd::expect_failure_and_text 'oc new-app --image-stream ruby https://github.com/openshift/rails-ex --dry-run' 'error: only a partial match was found for \"ruby\":'
 # when the tag is specified explicitly, the operation is successful
 os::cmd::expect_success 'oc new-app --image-stream ruby:2.4 https://github.com/openshift/rails-ex --dry-run'
-
 os::cmd::expect_success 'oc delete imagestreams --all'
 
+# newapp does not attempt to create an imagestream that already exists for a Docker image
+os::cmd::expect_success 'oc delete is ruby --ignore-not-found'
+os::cmd::expect_success 'oc delete is ruby-ex --ignore-not-found'
```

Comment:
we should know at this point in the test if these deletes are necessary or not.

Reply:
i'll apply the same remove / diagnose if something is amiss methodology

Reply:
tests pass with this removed .... will be part of next push